nyc_taxis/challenges/default.json

```diff
@@ -939,4 +939,12 @@
       "schedule": [
         {{ rally.collect(parts="common/update-aggs-only-schedule.json") }}
       ]
+    },
```

Comment:
Thank you for putting it to a separate file! It feels like nyc_taxis just keeps on growing :) it's less messy that way! I wonder if eventually we want to separate some other challenges out as well..